src/main/java/appeng/client/render/crafting/ItemEncodedPatternBakedModel.java

```diff
@@ -227,7 +227,7 @@ public IBakedModel handleItemState( IBakedModel originalModel, ItemStack stack,
 			{
 				ItemEncodedPattern iep = (ItemEncodedPattern) stack.getItem();
 				ItemStack output = iep.getOutput( stack );
-				if( output != null )
+				if( output != null && !output.isEmpty() )
```

Comment:
I'm probably to conservative here, but I'm not sure, whether `details.getCondensedOutputs()[0].getItemStack()` in `ItemEncodedPattern::getOutput(ItemStack)` might be null or not.